pkg/data/management/podadmissionconfigurationtemplate_data.go

```diff
@@ -29,6 +29,8 @@ var FeatureAppNS = []string{
 	"cattle-windows-gmsa-system", // Windows GMSA
 	"cattle-sriov-system",        // Sriov
 	"cattle-ui-plugin-system",    // UI Plugin System
+	"cis-operator-system",        // From RKE2 default restricted PSA Config. See https://github.com/rancher/rke2/blob/release-1.25/pkg/rke2/psa.go#L58
```

Comment:
Please change the link to permalink (with commit ID) instead of branch.

Reply:
fixed.